### PR TITLE
Update Cassandra limitations with racks, CNI, and security.

### PR DESCRIPTION
**Pull request by triclambert**

### CODE REVIEW

File: frameworks/cassandra/docs/limitations.md [hunk 1]
```diff
@@ -5,7 +5,19 @@ feature_maturity: preview
 enterprise: 'no'
 ---
 
-* Data Center Name
+## Rack Aware Placement
```
Comment (sascala) [inline frameworks/cassandra/docs/limitations.md]: Rack-Aware Placement

File: frameworks/cassandra/docs/limitations.md [hunk 2]
```diff
@@ -5,7 +5,19 @@ feature_maturity: preview
...
+
+The DC/OS Cassandra Service does not currently leverage rack information for placement decisions.  However, placement constraints can be used to manually place instances across racks if desired.   
+
+## Security
+
+The security features  are not supported at this time.
```
Comment (sascala) [inline frameworks/cassandra/docs/limitations.md]: extra space after "features"

**Comment by triclambert** [comment]
@sascala updated.

@keithchambers PTAL.

File: frameworks/cassandra/docs/limitations.md [hunk 3]
```diff
@@ -5,7 +5,19 @@ feature_maturity: preview
 enterprise: 'no'
 ---
 
-* Data Center Name
+## Rack-Aware Placement
+
+The DC/OS Cassandra Service does not currently leverage rack information for placement decisions.  However, placement across racks can be performed manually using placement constraints; application-level configuration will still not be possible.
```
Comment (keithchambers) [inline frameworks/cassandra/docs/limitations.md]: While technically correct, I can see how someone might read this and come away with an incorrect understanding. I recommend we KISS and state something like "Cassandra's Rack-Aware Replication is not supported at this time."

File: frameworks/cassandra/docs/limitations.md [hunk 4]
```diff
@@ -5,7 +5,19 @@ feature_maturity: preview
...
 ---
 
-* Data Center Name
+## Rack-Aware Placement
+
+The DC/OS Cassandra Service does not currently leverage rack information for placement decisions.  However, placement across racks can be performed manually using placement constraints; application-level configuration will still not be possible.   
+
+## Security
+
+The security features are not supported at this time.
```
Comment (keithchambers) [inline frameworks/cassandra/docs/limitations.md]: Same as above ... "Cassandra's native TLS, authentication, and authorization features are not supported at this time."

File: frameworks/cassandra/docs/limitations.md (hunk 3 above)
Comment (triclambert) [inline frameworks/cassandra/docs/limitations.md]: Ok.

**Comment by keithchambers** [comment]
LGTM as long as @sascala approves.

**Comment by sascala** [comment]
I suggested some tiny changes, but otherwise LGTM.
—

Suzanne Scala
Technical writer, Mesosphere

File: frameworks/cassandra/docs/limitations.md [hunk 5]
```diff
@@ -5,11 +5,23 @@ feature_maturity: preview
...
+
+Apache Cassandra's Rack-Aware Replication is not supported at this time.
+
+## Security
+
+Apache Cassandra's native TLS, authentication, and authorization features are not supported at this time.
+
+## Overlay networks
+
+When the DC/OS Cassandra Service is deployed on a virtual network, the configuration cannot be updated after installation.
```
Comment (mrbrowning) [inline frameworks/cassandra/docs/limitations.md]: The configuration here meaning which overlay network it belongs to, correct? That could be clarified.

**Pull request merged**